.github/workflows/dapr.yml

```diff
@@ -63,20 +65,26 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ env.GOVER }}
-      - name: Install Docker CE for buildx
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
On Linux hosts, we are building images for all archs, including arm64 and armv7. QEMU is needed to build those Docker images. I don’t think that was a problem for distroless, because with those we were just copying the binary; for Mariner, there is a Docker build step before.